src/main/resources/config/liquibase/changelog/20250506142300_changelog.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20250506142300" author="n0w0rk">
+        <!-- In this migration we want to remove duplicated course learner Profiles. -->
+
+        <!-- We select all -->
+        <sql dbms="mysql, postgresql, h2">
+            DELETE
+            FROM course_learner_profile
+            WHERE id in
+                  (SELECT id FROM
+                      (SELECT clp1.id
+                       FROM course_learner_profile clp1, course_learner_profile clp2
+                       WHERE clp1.course_id = clp2.course_id AND clp1.id &lt; clp2.id AND clp1.learner_profile_id = clp2.learner_profile_id)reference
```

Comment:
there seems to be a wrong character "&lt;"

Reply:
<img width="197" alt="image" src="https://github.com/user-attachments/assets/0dc1d0f4-0a57-450b-a27c-c367847c035f" />